src/util.ts

```diff
@@ -615,7 +615,7 @@ export class Util {
 
         if (!err || autoAuthFailed) {
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          let projectId = (authClient as any)._cachedProjectId;
```

Comment:
this would be a good time to get rid of the `as any` here, since that method should very much be public and exist on the types now

Reply:
Good call, done!